src/seedu/addressbook/data/person/AddressProperty.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Created by raynoldng on 5/2/18.
```

Comment:
We can get to know this from the commit history; as such, remove these kinda redundant/not-so-useful comments generated by the IDE.